app/javascript/providers/StudySearchProvider.js

```diff
@@ -104,7 +107,7 @@ export function PropsStudySearchProvider(props) {
 
     setSearchState({
       params,
-      isError: false,
+      isError: results.ok || results.ok == undefined ? false : !results.ok,
```

Comment:
if you're trying to set isError to `false` when results.ok is either false or undefined,    you can just use `!!results.ok`.  If you're doing something else, please add a comment so it's easier to read (and probably this is too complex to put in a one-liner)